src/components/layout/footer/disclaimer.tsx

```diff
@@ -16,15 +16,15 @@ type DisclaimerSectionProps = {
 }
 
 const DisclaimerSection = ({ is_academy = false }: DisclaimerSectionProps) => {
-    const { is_eu, is_non_eu } = useRegion()
+    const { is_eu, is_non_eu, is_africa } = useRegion()
     return (
         <>
             <DisclaimerWrapper>
-                {is_non_eu && (
+                {(is_non_eu || is_africa) && (
                     <>
                         <DisclaimerParagraph>
                             <Localize
-                                translate_text="In the EU, financial products are offered by Deriv Investments (Europe) Limited, W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR 9033, Malta, licensed in Malta (<0>licence no. IS/70156</0>) and regulated by the Malta Financial Services Authority, Triq l-Imdina, Zone 1, Central Business District, Birkirkara CBD 1010, Malta, under the Investments Services Act."
+                                translate_text="Deriv Investments (Europe) Limited is licensed and regulated by the Malta Financial Services Authority, Triq L-Imdina, Zone 1, Central Business District, Birkirkara CBD 1010, Malta, under the Investment Services Act (<0>licence</0>). The registered office of Deriv Investments (Europe) Limited is at W Business Centre, Level 3, Triq Dun Karm, Birkirkara BKR9033, Malta."
```

Comment:
Should we use the new translation key extraction method with **TString**? Since we were refactoring a few days ago

Reply:
This task have high priority and come to website faster. In this case no. Someone who refactor it should resolve conflicts and everything will fine :)